app/views/shared/_image_shares_modal.erb

```diff
@@ -0,0 +1,26 @@
+<div class="modal fade" id="imageSharesModal" tabindex="-1" role="dialog" aria-labelledby="exampleModalLabel" aria-hidden="true">
+  <div class="modal-dialog" role="document">
+    <div class="modal-content">
+      <div class="modal-header">
+        <h5 class="modal-title" id="exampleModalLabel">Share an Image</h5>
```

Comment:
Minor: can we update the `id` to make more sense?

Reply:
Certainly